src/wolff_producer.erl

```diff
@@ -130,6 +136,9 @@
 %%    exact max allowed message size configured in kafka.
 %% * `max_linger_ms': Age in milliseconds a baatch can stay in queue when the connection
 %%    is idle (as in no pending acks). Default: 0 (as in send immediately)
+%% * `max_linger_bytes': Number of bytes to collect before sending it to Kafka.
```

Comment:
Q: what is the difference between `max_linger_bytes` and `max_batch_bytes`?  Just looking at the descriptions, they seem the same to me.  :see_no_evil: 

From the code, it seems that this is the number of bytes that are collected in the producer mailbox before it's enqueued (to disk or memory), before even sending to kafka?

Reply:
yes, max_batch_bytes is for the popping end of the queue.
max_linger_bytes is at the pushing end.